duckduckgo/iOS
# [Xcode] adds script for updating Settings bundle

![duckduckgo-settings](https://user-images.githubusercontent.com/276137/69600785-eb6af380-0fdf-11ea-975b-593734261745.png)
<hr/>
Task/Issue URL: #497 

**Description**:
This adds:
 1. a config file that defines `MARKETING_VERSION`–`Version.xcconfig`;
 2. a settings bundle–`Settings.bundle`;
 3. a script file–`set_version.sh`.

The script file pulls `MARKETING_VERSION` from the config file, and writes it to the settings bundle.

**Steps to test this PR**:
1. Run `chmod +x set_version.sh` to ensure it is executable;
2. Run `set_version.sh 7.33.1`

**Known Issues**:
This script should be run prior to build the app for a release, and the resulting changes committed.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
